src/interfaces/IAppliance.js

```diff
@@ -0,0 +1,95 @@
+import {
+	InterfaceInstantiationError,
+	NotImplementedError,
+} from '../errors'
+
+class IAppliance {
+	/**
+	 * Constructor for a new appliance
+	 * @param  {Object} overrideSettings Values to override default appliance settings.
```

Comment:
Quibble: Insert newline between description and param list. (And add period to end of description.)